Heroku friendly seeds

DrewProebstel opened the pull request:
Please include a summary of the changes and/or which issue is fixed. Please also include relevant motivation and context. 

Creates seeds that do not use faker. Get's rid of old tests that do not reflect current db
Were there any issues that arose? 

## Issues
Closes #

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactor feature (non-breaking change that modifies existing work)
- [ ] Cosmetic changes (changes to layout or visuals in CSS file)

# How Has This Been Tested?
- [ ] React Dev Tools
- [ ] Functionality check through web browser
- [ ] Ran Cypress test suite

# Checklist:
- [ ] My code follows the style guidelines of this project (semicolons, indentation, naming conventions, etc..).
- [ ] I have performed a self-review of my code.
- [ ] I have deleted all comments and console.logs after functionality and understanding is final.
- [ ] I have added any issues closed by this PR in the Issues section.
- [ ] I have added others as reviewers (when applicable).

DrewProebstel committed:
movie serive and poro added

DrewProebstel committed:
seeding pulls movies from api and saves them to the database

DrewProebstel committed:
remove summary from movie table

DrewProebstel committed:
wip

DrewProebstel committed:
Merge branch 'set_up_seeds_for_movies' into get_chunk

DrewProebstel committed:
wip

DrewProebstel committed:
hitting pry

DrewProebstel committed:
adds test chuck redefines some schema

DrewProebstel committed:
removes a pry

DrewProebstel committed:
db seeds properly

DrewProebstel committed:
removed unneeded tests and set votes to default of 0

erock02 commented on Gemfile.lock:
```diff
@@ -210,4 +217,4 @@ RUBY VERSION
    ruby 2.7.4p191
 
 BUNDLED WITH
-   2.3.14
```
Why the bundle version change?

erock02 committed:
Update Gemfile.lock

changing bundle version to be 2.3.14 to make sure these changes can get pushed to Heroku

erock02 committed:
Delete .resultset.json

We don't need coverage files.

erock02 commented on db/migrate/20220830145530_create_events.rb:
```diff
@@ -1,7 +1,7 @@
 class CreateEvents < ActiveRecord::Migration[5.2]
   def change
     create_table :events do |t|
-      t.datetime :date
```
think we should leave this as datetime since datetime still returns a string. We can discuss this

erock02 committed:
Update 20220830145530_create_events.rb

erock02 committed:
Update schema.rb

leaving data as datetime
